fcrepo-serialization/src/test/java/org/fcrepo/camel/serialization/BinaryDisabledRouteTest.java

```diff
@@ -146,7 +147,7 @@ public void configure() throws Exception {
         });
         context.start();
         // this should be zero because writing binaries is disabled by default.
-        getMockEndpoint("mock:direct:binary_file").expectedMessageCount(0);
+        getMockEndpoint("mock:file:mock:direct:binary_file").expectedMessageCount(0);
```

Comment:
I see what I was doing wrong now that was forcing me to do that convoluted mock: string.